ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -645,6 +651,87 @@ class ClientQueryViewController: UITableViewController, Themeable {
 		}
 
 	}
+
+	func upload(itemURL: URL, name: String, viewDidAppearHandler: ClientActionVieDidAppearHandler? = nil, completionHandler: ClientActionCompletionHandler? = nil) {
```

Comment:
Good catch, just copy-paste from another action, I think even that we don't need this for the uploads because it's not presenting any screen in this step.

I'll remove it from the method definition.